internal/releaseauth/testdata/sample.private.key

```diff
@@ -0,0 +1,106 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
I had the exact same thought and I have someone from product security that is reviewing this change for their purposes. I'm sure they will chime in if this could raise an alarm.

Reply:
We do have scanners, but its not a huge deal. We can just mark it as a false positive.